cypress.config.js

```diff
@@ -2,6 +2,10 @@ const { defineConfig } = require('cypress');
 
 module.exports = defineConfig({
   e2e: {
+    env: {
+      EMAIL: 'miretta2@ukr.net',
+      PASSWORD: 'Miretta29+'
```

Comment:
Avoid hardcoding sensitive data such as email and password directly in your code. This can lead to security issues, especially if the code is publicly accessible. Consider using environment variables for these. You can use the `dotenv` package to load environment variables from a `.env` file in your project. This file should not be committed to your repository. Instead, provide a `.env.example` file with the necessary environment variables listed, but without the actual values.